[2024-04-08T01:01:26.417+0530] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_fetch_pipeline.select_files manual__2024-03-10T17:44:53+00:00 [queued]>
[2024-04-08T01:01:26.426+0530] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_fetch_pipeline.select_files manual__2024-03-10T17:44:53+00:00 [queued]>
[2024-04-08T01:01:26.426+0530] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-04-08T01:01:26.445+0530] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): select_files> on 2024-03-10 17:44:53+00:00
[2024-04-08T01:01:26.451+0530] {standard_task_runner.py:60} INFO - Started process 39150 to run task
[2024-04-08T01:01:26.454+0530] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'data_fetch_pipeline', 'select_files', 'manual__2024-03-10T17:44:53+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/data_fetch_pipeline.py', '--cfg-path', '/tmp/tmperzmj43b']
[2024-04-08T01:01:26.456+0530] {standard_task_runner.py:88} INFO - Job 60: Subtask select_files
[2024-04-08T01:01:26.501+0530] {task_command.py:423} INFO - Running <TaskInstance: data_fetch_pipeline.select_files manual__2024-03-10T17:44:53+00:00 [running]> on host LAPTOP-HO799B2R.
[2024-04-08T01:01:26.592+0530] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='data_fetch_pipeline' AIRFLOW_CTX_TASK_ID='select_files' AIRFLOW_CTX_EXECUTION_DATE='2024-03-10T17:44:53+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-10T17:44:53+00:00'
[2024-04-08T01:01:26.594+0530] {data_fetch_pipeline.py:65} ERROR - Error selecting files for year 2022: '<=' not supported between instances of 'int' and 'str'
[2024-04-08T01:01:26.595+0530] {python.py:201} INFO - Done. Returned value was: None
[2024-04-08T01:01:26.602+0530] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=data_fetch_pipeline, task_id=select_files, execution_date=20240310T174453, start_date=20240407T193126, end_date=20240407T193126
[2024-04-08T01:01:26.629+0530] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-08T01:01:26.658+0530] {taskinstance.py:3280} INFO - 6 downstream tasks scheduled from follow-on schedule check
